s3parq/publish_parq.py

```diff
@@ -170,6 +184,7 @@ def publish(bucket: str, key: str, partitions: iter, dataframe: pd.DataFrame, re
         ARGS: 
             Schema_name: str
             Table_name: str
+            iam_role
```

Comment:
Please add the :str here and lowercase schema_name and table_name. It's so inconsequential but. It's needed.

Reply:
fixed